package.json

```diff
@@ -45,7 +45,6 @@
     "husky": "=4",
     "lerna": "^6.4.1",
     "lint-staged": ">=10",
-    "plop": "^2.7.4",
```

Comment:
I think you can also remove the generate command https://github.com/strapi/design-system/blob/014fa87b3c4917b00ddeb5afccb856f8ed66bc5a/package.json#L17